pydoctor/templates/apidocs.css

```diff
@@ -18,7 +18,7 @@
 .categoryHeader {
     font-size: 24px;
     color: #777;
-    margin-bottom: 1.8em;
+    margin-bottom: 0.5em;
```

Comment:
I like having a bit more margin between different parts of the document. Rather than reducing the margin here, we could increase margins elsewhere.

This is subjective of course, but in any case this change is not necessary to improve the code fragments layout.